dask/array/tests/test_array_function.py

```diff
@@ -191,16 +180,12 @@ def test_non_existent_func():
     # Regression test for __array_function__ becoming default in numpy 1.17
     # dask has no sort function, so ensure that this still calls np.sort
     x = da.from_array(np.array([1, 2, 4, 3]), chunks=(2,))
-    if IS_NEP18_ACTIVE:
-        with pytest.warns(
-            FutureWarning, match="The `numpy.sort` function is not implemented by Dask"
-        ):
-            assert list(np.sort(x)) == [1, 2, 3, 4]
-    else:
+    with pytest.warns(
+        FutureWarning, match="The `numpy.sort` function is not implemented by Dask"
+    ):
         assert list(np.sort(x)) == [1, 2, 3, 4]
```

Comment:
Left the comment above in and kept the test as these seem to still make sense. Though please let me know if not